package.json

```diff
@@ -45,22 +43,26 @@
     "gh-pages": "^2.0.0",
     "react": "^16.4.1",
     "react-dom": "^16.4.1",
-    "react-scripts": "^1.1.4",
+    "react-scripts": "^2.1.3",
     "react-testing-library": "^5.1.0",
-    "rollup": "^0.64.1",
-    "rollup-plugin-babel": "^3.0.7",
-    "rollup-plugin-commonjs": "^9.1.3",
-    "rollup-plugin-node-resolve": "^3.3.0",
+    "rollup": "^1.0.2",
+    "rollup-plugin-babel": "^4.2.0",
+    "rollup-plugin-commonjs": "^9.2.0",
+    "rollup-plugin-node-resolve": "^4.0.0",
     "rollup-plugin-peer-deps-external": "^2.2.0",
-    "rollup-plugin-postcss": "^1.6.2",
-    "rollup-plugin-url": "^1.4.0"
+    "rollup-plugin-postcss": "^1.6.3",
+    "rollup-plugin-url": "^2.1.0"
   },
   "files": [
     "dist"
   ],
   "dependencies": {
     "copy-to-clipboard": "^3.0.8",
     "create-react-context": "^0.2.3",
+    "prop-types": "^15.6.2",
     "react-element-to-jsx-string": "^14.0.1"
+  },
+  "resolutions": {
+    "eslint": "5.6.0"
```

Comment:
this isn't the most recent version, but create-react-app's scripts warned if they resolved to a newer version than this.